archive/har.d

```diff
@@ -297,21 +317,21 @@ struct FileProperties
         {
             import std.string : indexOf;
             bool useQuotes = false;
-            foreach(arg; args)
+            foreach (arg; args)
```

Comment:
always a good idea to split out in different commits formatting-only changes (eg using dfmt) vs actual code changes (unless https://github.com/dlang-community/dfmt/issues/159 is implemented)

Reply:
I agree in most cases, but I'm fine with it in the scenario